PCbuild/regen.targets

```diff
@@ -82,31 +82,31 @@
   <Target Name="_RegenGlobalObjects"
           DependsOnTargets="FindPythonForBuild">
     <Message Text="Regenerate Global Objects" Importance="high" />
-    <Exec Command="$(PythonForBuild) Tools\build\generate_global_objects.py"
+    <Exec Command="$(PythonForBuild) &quot;Tools\build\generate_global_objects.py&quot;"
           WorkingDirectory="$(PySourcePath)" />
   </Target>
 
   <Target Name="_RegenCases"
           Inputs="@(_CasesSources)" Outputs="@(_CasesOutputs)"
           DependsOnTargets="FindPythonForBuild">
     <Message Text="Regenerate cases" Importance="high" />
-    <Exec Command="$(PythonForBuild) $(PySourcePath)Tools\cases_generator\opcode_id_generator.py $(PySourcePath)Python\bytecodes.c"
+    <Exec Command="$(PythonForBuild) &quot;$(PySourcePath)Tools\cases_generator\opcode_id_generator.py&quot; &quot;$(PySourcePath)Python\bytecodes.c&quot;"
```

Comment:
By my reading, these are the only ones that need it.

It might also be worth testing whether removing the `$(PySourcePath)` variable is enough, as we're already explicitly in that working directory.

Reply:
Yeah I think that is better.

Reply:
```suggestion
    <Exec Command="$(PythonForBuild) Tools\cases_generator\opcode_id_generator.py Python\bytecodes.c"
```

Reply:
I have just checked and that also works.